sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -255,12 +257,21 @@ private void forceFlush() {
       exportBatches(spansCopy);
     }
 
+    @SuppressWarnings("BooleanParameter")
```

Comment:
was this yelling for usage of the AtomicBoolean? :(

Reply:
Yeah :(